cmd/minikube/cmd/config/config.go

```diff
@@ -91,6 +91,14 @@ var settings = []Setting{
 		name: "kubernetes-version",
 		set:  SetString,
 	},
+	{
+		name: "dashboard",
+		set:  SetBool,
+	},
+	{
+		name: "addon-manager",
+		set:  SetBool,
```

Comment:
For now I have put this code in the command itself as setFn takes (string, string).  I definitely think this is worth doing though to minimize strange states where people could perhaps be setting the config w/ minikube config, outside of minikube addon.